web/gds-admin-ui/src/pages/app/details/BasicDetails/components/TrisaImplementationDetailsForm.js

```diff
@@ -0,0 +1,106 @@
+import { yupResolver } from '@hookform/resolvers/yup'
+import Field from 'components/Field'
+import { ModalCloseButton, ModalContext } from 'components/Modal'
+import { Alert, Button, Col, Form, Row } from 'react-bootstrap'
+import { useForm } from 'react-hook-form'
+import { getTrisaImplementationDetailsInitialValue } from 'utils/form-references'
+import React from 'react'
+import * as yup from 'yup'
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import useSafeDispatch from 'hooks/useSafeDispatch'
+import { clearTrisaImplementationDetailsErrorMessage, updateTrisaImplementationDetailsResponse } from 'redux/vasp-details'
+import { getTrisaDetailsErrorState } from 'redux/selectors'
+
+const trisaEndpointPattern = /([a-zA-Z0-9.]+):((?!(0))[0-9]+)/
+
+const schema = yup.object().shape({
+    trisa_endpoint: yup.string().trim().matches(trisaEndpointPattern, "trisa endpoint is not valid"),
+    common_name: yup.string().trim().test("should-match", "common name should match the TRISA endpoint without the port", (value, context) => {
```

Comment:
This is true 98% of the time, would it be possible to change this to a warning instead of an error and still allow the user to submit? Just for the 2% of the time where this is not true.

Reply:
yes